<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editTitleDialog</class>
 <widget class="QDialog" name="editTitleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Subtitle</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>End:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_7">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_8">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Text:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
